src/components/SpecialResolution/CreateSpecialResolutionSummary.vue

```diff
@@ -0,0 +1,209 @@
+                </li>
+                <li class="mt-4">
+                  A printed copy of the signed special resolution (Form 06 COO) has been retained with the Cooperative
+                  Association's records.
+                </li>
+              </ul>
+            </v-form>
+          </v-col>
+        </v-row>
+      </div>
+    </section>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Mixins, Watch } from 'vue-property-decorator'
+import { Action, Getter } from 'vuex-class'
+import {
+  ActionBindingIF,
+  ResourceIF,
+  FormIF,
+  CreateResolutionIF,
+  EntitySnapshotIF
+} from '@/interfaces/'
+import { CommonMixin, DateMixin } from '@/mixins/'
+import { HelpSection } from '@/components/common/'
+import { DatePicker as DatePickerShared } from '@bcrs-shared-components/date-picker/'
+
+@Component({
+  components: {
+    HelpSection,
+    DatePickerShared
+  }
+})
+export default class CreateSpecialResolutionSummary extends Mixins(CommonMixin, DateMixin) {
+  @Getter getcreateResolution!: CreateResolutionIF;
+  @Getter getAppValidate!: boolean;
+  @Getter getSpecialResolutionConfirmValid!: boolean;
+  @Getter getEntitySnapshot!: EntitySnapshotIF
+
+  @Action setResolution!: ActionBindingIF;
+  @Action setSpecialResolutionConfirmStateValidity!: ActionBindingIF;
+
+  // Refs
+  $refs!: {
+    confirmResolutionChkFormRef: FormIF;
+  };
+
+  protected resolutionConfirmed = false;
+
+  /** Validation rule for checkbox. */
+  protected confirmCompletionResolution = [
+    v => {
+      return !!v
+    }
+  ];
+
+  protected async onResolutionConfirmedChange (resolutionConfirmed: boolean): Promise<void> {
+    // This is required as there are timing issues between this component and the CompleteResolutionSummary
+    // component.  The CompleteResolutionSummary isn't always able to detect that the confirm checkbox
+    // value has changed without using nextTick()
+    await this.$nextTick()
+    this.setResolution({
+      ...this.getcreateResolution,
+      resolutionConfirmed: resolutionConfirmed
+    })
+    this.setSpecialResolutionConfirmStateValidity(resolutionConfirmed)
+  }
+
+  /** The name section validity state (when prompted by app). */
+  get invalidSpecialResolutionConfirmSection (): boolean {
+    return this.getAppValidate && !this.getSpecialResolutionConfirmValid
+  }
+  /** The resolution date in readable. */
+  get resolutionDateText (): string {
+    return this.yyyyMmDdToPacificDate(this.getcreateResolution.resolutionDate, true)
+  }
+
+  /** The signing date in readable. */
+  get signingDate (): string {
+    return this.yyyyMmDdToPacificDate(this.getcreateResolution.signingDate, true)
+  }
+
+  /** The signing person details. */
+  get signingParty (): string {
+    if (this.getcreateResolution.signatory) {
+      const { givenName, additionalName = '', familyName } = this.getcreateResolution.signatory
+
+      return `${givenName} ${additionalName || ''} ${familyName}`
```

Comment:
1. additional name should not be falsy since you default it on line 162
2. does it matter that you could have 2 spaces between given name and family name?